bestbook/views/__init__.py

```diff
@@ -240,3 +269,37 @@ def get(self):
 class Admin(MethodView):
     def get(self):
         return render_template("base.html", template="admin.html", models=models)
+
+
+class RecommendationApproval(MethodView):
+    def get(self):
+        return render_template("base.html", template="approve-recommendation.html", models={
+            "recommendations": Recommendation,
+            "books": Book,
+            "topics": Topic
+        })
+
+class RequestApproval(MethodView):
+    def get(self):
+        return render_template("base.html", template="approve-request.html", models={
+            "requests": Request,
+            "topics": Topic
+        })
+
+
+def fetch_work(work_olid):
+    base_url = "https://openlibrary.org/works/"
+    url = "{}{}.json".format(base_url, work_olid)
+    resp = requests.get(url)
+
+    result = {}
+
+    if resp.status_code == 200:
+        data = resp.json()
+        result['title'] = data['title']
+    else:
+        result['title'] = work_olid
+
+    result['link'] = "{}{}".format(base_url, work_olid)
+
+    return result
```

Comment:
This function doesn't belong in this file.  Suggestions are welcome.

Reply:
We may want to put this in the apis/books.py file attached to the Book class as a `classmethod`